requirements/base.txt

```diff
@@ -103,21 +103,21 @@ pylint==2.12.2
     #   pylint-plugin-utils
 pylint-celery==0.3
     # via edx-lint
-pylint-django==2.5.0
+pylint-django==2.5.2
     # via edx-lint
 pylint-plugin-utils==0.7
     # via
     #   pylint-celery
     #   pylint-django
-pymongo==4.0.1
+pymongo==3.12.3
```

Comment:
this is a major version downgrade, and I don't understand why this is being reflected almost everywhere